flink-streaming-scala/src/main/scala/org/apache/flink/streaming/api/scala/KeyedStream.scala

```diff
@@ -66,9 +66,9 @@ class KeyedStream[T, K](javaStream: KeyedJavaStream[T, K]) extends DataStream[T]
     * function, this function can also query the time and set timers. When reacting to the firing
     * of set timers the function can directly emit elements and/or register yet more timers.
     *
-    * @param processFunction The [[ProcessFunction]] that is called for each element
-    *                   in the stream.
+    * @param processFunction The [[ProcessFunction]] that is called for each element in the stream.
```

Comment:
Please also add that the user now should use the new `KeyedProcessFunction` instead.